docs/topics/signals.txt

```diff
@@ -277,7 +277,10 @@ Disconnecting signals
 
 To disconnect a receiver from a signal, call :meth:`Signal.disconnect`. The
 arguments are as described in :meth:`.Signal.connect`. The method returns
-``True`` if a receiver was disconnected and ``False`` if not.
+``True`` if a receiver was disconnected and ``False`` if not. This only applies
+when passing a model class to ``sender`` (``sender=MyModel``). Using a lazy
+reference (``sender='myapp.MyModel'``) will always return ``None``,
+independently if the receiver was disconnected or not.
```

Comment:
I would say, "always returns `None`, independent of whether the receiver was disconnected." ("always returns" matches the tense of "the method returns" in the previous sentence.)

Reply:
@cjerdonek I've pushed edits.